src/lib.rs

```diff
@@ -78,6 +78,79 @@
 //! ### Support
 //!
 //! Please refer to the DID specification: [DID](https://www.w3.org/TR/did-core/)
+
+mod resolver;
 pub mod rpc;
 pub mod types;
 mod util;
+
+use serde::Deserialize;
+use std::str::FromStr;
+
+use anyhow::Result;
+use ethers::types::Address;
+use jsonrpsee::server::Server;
+
+pub use crate::{
+    resolver::{did_registry, Resolver},
+    rpc::DidRegistryMethods,
+    rpc::DidRegistryServer,
+};
+
+const DEFAULT_ADDRESS: &str = "127.0.0.1:9944";
+const DEFAULT_PROVIDER: &str = "http://127.0.0.1:8545";
+
+// TODO: Get registry address from environment variable, or configuration file
+// in order to support multiple chains, we may need to support multiple providers via RPC
+// so it could be worth defining a config file that maps chainId to RPC provider (like
+// did-eth-resolver)
+/// The address of the DID Registry contract on the Ethereum Sepolia Testnet
+pub const DID_ETH_REGISTRY: &str = "0xd1D374DDE031075157fDb64536eF5cC13Ae75000";
```

Comment:
This will be a proxy contract with a stable address on all networks.   Allowing override probably makes sense, but may also create a security loophole.

Reply:
we should be explicit that the override is only for tests, or restrict overriding the tests only when we're sure we're in a test environment